src/components/SubmitButton.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import "../styles/Button.css";
+
+function CustomButton({ uploadOnClick }) {
```

Comment:
CustomButton is nondescript, we should write self-documenting code.
Rename it to something that descripts its function e.q SubmitButton